Zend/tests/type_declarations/scalar_reserved2_class_alias.phpt

```diff
@@ -7,4 +7,4 @@ class foobar {}
 class_alias("foobar", "int");
 ?>
 --EXPECTF--
-Fatal error: Cannot use 'int' as class name as it is reserved in %s on line %d
+Fatal error: Cannot use 'int' as a type alias as it is reserved in %s on line %d
```

Comment:
Possibly? But `type alias` is also fine by me.
```suggestion
Fatal error: Cannot use 'int' as a class alias as it is reserved in %s on line %d
```

Reply:
Class alias is definitely better. There are no such elements as type aliases in PHP.

Reply:
Hmm, yeah, I think I misunderstood where this got applied, fix at #15996